webpack.config.js

```diff
@@ -3,20 +3,27 @@ const mode = process.env.NODE_ENV || 'production';
 const BundleAnalyzerPlugin = require( 'webpack-bundle-analyzer' )
 	.BundleAnalyzerPlugin;
 const EnvironmentPlugin = require( 'webpack' ).EnvironmentPlugin;
+const DynamicPublicPathPlugin = require( 'dynamic-public-path-webpack-plugin' );
+const SriPlugin = require( 'webpack-subresource-integrity' );
+const ManifestPlugin = require( 'webpack-manifest-plugin' );
+const { CleanWebpackPlugin } = require( 'clean-webpack-plugin' );
 
 const sharedConfig = {
 	mode: mode,
 	entry: {
 		'features/titles': path.resolve( __dirname, 'src/features/titles/index.js' ),
+		'features/blocks/personalization': path.resolve( __dirname, 'inc/features/blocks/personalization/index.js' ),
+		'features/blocks/personalization-variant': path.resolve( __dirname, 'inc/features/blocks/personalization-variant/index.js' ),
```

Comment:
Seems like the JS should be in `src/features` not `inc/features` for consistency?

Reply:
I was initially going to do that but it's not how we build any of our other blocks in other projects. Everything is colocated usually. I know what you mean though given that the Ab test feature code is in `src`.

Reply:
Yeah, I think it makes sense to be consistent within this project; if we prefer inc, let's consider changing A/B to match.

Reply:
I think we can reorganise it properly when we merge this repo into the analytics plugin repo.